docs/index.rst

```diff
@@ -10,10 +10,11 @@ The fairlearn project seeks to enable anyone involved in the development of arti
 The code is `available from GitHub <https://github.com/fairlearn/fairlearn>`_.
 
 .. toctree::
+   :maxdepth: 2
 
-    fairlearn.exceptions
-    fairlearn.metrics
-    fairlearn.postprocessing
-    fairlearn.reductions
-    fairlearn.widget
-
+   auto_examples/binary_classification_COMPAS
```

Comment:
> The auto_generated examples doesn't exist when the index.rst is processed, and therefore gives an error.

The generated example is `auto_example/plot_binary_classification_COMPAS`... :)

Reply:
thanks :)